cli.js

```diff
@@ -13,6 +13,7 @@ const cli = meow(`
 
 	Options
 	  --github -g  Open the GitHub repo of the package
+		--yarn -y    Open the Yarn homepage of the package
```

Comment:
Incorrect indentation here.

And you need to define `-y` in the below `alias` definition.